setup.cfg

```diff
@@ -1,3 +1,16 @@
 [bdist_rpm]
 requires=numpy pyresample pyorbital
 release=1
+
+# See the docstring in versioneer.py for instructions. Note that you must
+# re-run 'versioneer.py setup' after changing this section, and commit the
+# resulting files.
+
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = trollsched/version.py
+versionfile_build = 
+tag_prefix = v
+parentdir_prefix =
```

Comment:
If you run coverage (or flake8) you may want to cover the settings from satpy. They include telling coverage to ignore the versioneer files.

Reply:
Thanks.

Reply:
Doesn't look like you actually did this. Do you still want to add it?

Reply:
Strange, it is in, indeed